QUICKSTART.md

```diff
@@ -402,6 +403,43 @@ tracerProvider.addSpanProcessor(BatchSpanProcessor.newBuilder(
 ).build());
 ```
 
+### TraceConfig
+
+`TraceConfig` associated with `TracerSdkProvider` can be updated via system properties, 
+environment variables and builder `set*` methods.  
+
+```java
+// Get TraceConfig associated with TracerSdkProvider 
+TraceConfig traceConfig = OpenTelemetrySdk.getTracerProvider().getActiveTraceConfig();
+
+// Get TraceConfig Builder
+Builder builder = traceConfig.toBuilder();
+
+// Read configuration options from system properties
+builder.readSystemProperties();
+
+// Read configuration options from environment variables
+builder.readEnvironmentVariables()
+
+// Set options via builder.set* methods, e.g.
+builder.setMaxNumberOfLinks(10);
+
+// Update the resulting TraceConfig instance
+OpenTelemetrySdk.getTracerProvider().updateActiveTraceConfig(builder.build());
+```
+
+Supported system properties and environment variables:
+
+| System property                  | Environment variable             | Purpose                                            | 
+|----------------------------------|----------------------------------|----------------------------------------------------|       
+| otel.config.sampler.probability  | OTEL_CONFIG_SAMPLER_PROBABILITY  | Sampler which is used when constructing a new span |                        
+| otel.config.max.attrs            | OTEL_CONFIG_MAX_ATTRS            | Max number of attributes per span                  |                        
```

Comment:
For all MAX attributes it would be useful to say what happens if that limit is exceeded.

Reply:
javadoc doesn't say what happens, I suppose exceeded should be dropped.

Reply:
Yes, extra attributes/links/events will be dropped. Too long values will be truncated.

Reply:
updated